tools/benchmark-cli/README.md

```diff
@@ -1,6 +1,7 @@
 ### Benchmark CLI
 
 #### Build
+The build process will require Gradle 3.3 and OpenJDK 1.8.
```

Comment:
Maybe instead of documenting this, we should change the documentation asking people to build this tool from the root of the project so the `gradlew` script is used and this is all automatically working fine?

That said, I'm not on the LS team anymore so maybe @danhermann or @jsvd will have more to say here? :)